util/config/config.go

```diff
@@ -207,12 +207,13 @@ var (
 		LogPath:                      "Log",
 		ChainDBPath:                  "ChainDB",
 		WalletFile:                   "wallet.json",
-		HttpWssDomain:                "{{.DashedIP}}.ipv4.nknlabs.io",
-		HttpWssCert:                  "certs/STAR.ipv4.nknlabs.io.cert",
-		HttpWssKey:                   "certs/STAR.ipv4.nknlabs.io.key",
-		HttpsJsonDomain:              "{{.DashedIP}}.ipv4.nknlabs.io",
-		HttpsJsonCert:                "certs/STAR.ipv4.nknlabs.io.cert",
-		HttpsJsonKey:                 "certs/STAR.ipv4.nknlabs.io.key",
+		DefaultTlsDomainTmpl:         "{{.DashedIP}}.ipv4.nknlabs.io",
+		DefaultTlsCert:               "certs/nknd.cert",
+		DefaultTlsKey:                "certs/nknd.key",
+		ACMEUserFile:                 "certs/user.json",
+		ACMEResourceFile:             "certs/resource.json",
```

Comment:
It's better to include domain in these filenames: `DefaultTlsCert`, `DefaultTlsKey`, `ACMEUserFile`, `ACMEResourceFile`. This way nknd don't need to re-obtain certificate if IP is changed from A to B and then back to A.